internal/pkg/deploy/cloudformation/env_test.go

```diff
@@ -100,8 +101,10 @@ func TestCloudFormation_UpgradeEnvironment(t *testing.T) {
 					})
 				})
 				s3 := mocks.NewMocks3Client(ctrl)
-				s3.EXPECT().Upload("mockbucket", "manual/templates/phonetool-test/6806328b58e482e354cfd1879fecc310113fcf1b95c6bdf058c486f440087847.yml", gomock.Any()).Return("url", nil)
-
+				s3.EXPECT().Upload("mockbucket", gomock.Any(), gomock.Any()).DoAndReturn(func(bucket, key string, data io.Reader) (string, error) {
```

Comment:
The previous way of doing it (with the `6806328b58e482e354cfd1879fecc310113fcf1b95c6bdf058c486f440087847`) has its merit of testing that we are using the sha as the name. But this LGTM as well. Just adding my thoughts here!